rest-api/src/main/java/org/jboss/pnc/client/RemoteResourceException.java

```diff
@@ -17,33 +17,45 @@
  */
 package org.jboss.pnc.client;
 
+import org.jboss.pnc.dto.response.ErrorResponse;
+
 import javax.ws.rs.ClientErrorException;
-import javax.ws.rs.core.Response;
+
+import lombok.Getter;
 
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
 public class RemoteResourceException extends ClientException {
 
+    @Getter
     private final int status;
 
+    @Getter
+    private final ErrorResponse response;
+
     public RemoteResourceException(Throwable cause) {
         super(cause);
-        status = -1;
+        this.status = -1;
+        this.response = null;
     }
 
     public RemoteResourceException(ClientErrorException cause) {
         super(cause);
-        Response response = cause.getResponse();
-        status = response.getStatus();
+        this.status = cause.getResponse().getStatus();
```

Comment:
is this NPE prof ? Is getResponse() always not null ?

Reply:
I just put it on one line, I didn't change behaviour :)

But it should be always set AFAIK.